nucliadb/src/nucliadb/search/search/results_hydrator/base.py

```diff
@@ -167,36 +171,75 @@ async def hydrate_text_block(
     )
 
 
+@hydrator_observer.wrap({"type": "text_block"})
+async def new_hydrate_text_block(
```

Comment:
where is this used?

Reply:
It'll be used later, I can remove it if it bothers you